commandbased/flowcontrol.py

```diff
@@ -0,0 +1,305 @@
+from wpilib.command.command import Command
+from wpilib.command.commandgroup import CommandGroup
+from commandbased.conditionalcommand import ConditionalCommand
+from commandbased.cancelcommand import CancelCommand
+
+import inspect
+
+__all__ = ['IF', 'ELIF', 'ELSE', 'WHILE', 'RETURN', 'BREAK']
+
+'''
+These functions can be used to make programming CommandGroups much more
+intuitive. For more information, check each method's docstring.
+'''
```

Comment:
This is meant to be a module docstring, right? This needs to be the first statement to be a docstring.

Also, this is just me being nitpicky, but [PEP 257](https://www.python.org/dev/peps/pep-0257/) says that docstrings conventionally use double quotes rather than single quotes.